<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutPutSpeakerDialog</class>
 <widget class="QDialog" name="OutPutSpeakerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>导出数据集</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>131</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>添加音频文件前缀</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sample_rate">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_geshi">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>171</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>音频文件采样率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>输出文件层级和标注文件格式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_qianzhui">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发音人</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_speaker">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>预设</string>
     </property>
     <widget class="QPushButton" name="pushButton_read_preinstall">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>读取所选预设</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_preinstall">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>211</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>声道数</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_channel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto_skip">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>是</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动跳过含有非中文的数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_guiyihua">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>归一化（留空为不进行归一化）
范围为-70~-5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_guiyihua">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-23</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>501</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>预览</string>
     </property>
     <widget class="QLabel" name="label_cengji">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>sounds
    - 1.wav
    - 2.wav
    ......
labels.txt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_geshi">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>231</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>1.wav|你好世界
2.wav|世界你好
......</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>文件层级：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>26</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>标注文件内容：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>预计导出文件数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_file_num">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
